### DISCUSSION TITLE
Scheduled deployments

### DISCUSSION THREAD

**Discussion by joostmeijles** [vercel/next.js#11277] [Help]
What is the recommended way for creating scheduled deployments (e.g. daily) for NextJS and Zeit?

NB. did not find any official Zapier, Github actions integrations.

**Comment by brianlovin** [comment]
I believe this is possible with Actions. Here's how I set mine up to re-deploy every Wednesday: https://github.com/specfm/design-details/blob/master/.github/workflows/deploy.yml

You'll need to get the `org-id` and `project-id` from the hidden `.now/project.json` file that gets generated whenever you link a local project to a project on ZEIT.

**Reply by timneutkens** [reply]
If you just want to redeploy you can use deploy hooks: https://zeit.co/docs/v2/more/deploy-hooks

**Reply by joostmeijles** [reply]
Thanks. Also need a rebuild, so copied and have now this: https://github.com/joostmeijles/blog/blob/master/.github/workflows/daily-deploy.yml
